homeassistant/components/shelly/climate.py

```diff
@@ -82,6 +81,7 @@ async def async_setup_climate_entities(
             sensor_block = block
 
     if sensor_block and device_block:
+        _LOGGER.info("Setup online climate device %s", wrapper.name)
```

Comment:
Normally we have these kind of messages at debug level.

Reply:
```suggestion
        _LOGGER.debug("Setup online climate device %s", wrapper.name)
```

Reply:
Yeah agree that it's a bit of a weird logging, as we also afterwards add the entity so see the data then?